test/libsolidity/syntaxTests/indexing/fixedbytes_noninteger_index.sol

```diff
@@ -0,0 +1,6 @@
+contract C {
+  function f() public {
+    bytes32 b;
+    b[888888888888888888888888888888888888888888888888888888888888888888888888888888888888888888888888888888888888888];
+  }
+}
```

Comment:
This is failing with an assertion, to be fixed by this PR.

Reply:
Done.